tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from parameterized import parameterized
+from openvino import inference_engine
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+    @parameterized.expand(['FP16', 'FP32'])
+    def test_load_intel(self, p):
+        face_detection = omz.Model('face-detection-0100')
+        xml_path, bin_path = face_detection.get_ir(precision=p)
+        
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+
+        self.assertEqual(os.path.basename(xml_path), 'face-detection-0100.xml')
+        self.assertEqual(os.path.basename(bin_path), 'face-detection-0100.bin')
+
+        self.assertEqual(os.path.basename(os.path.dirname(xml_path)), p)
+        self.assertEqual(os.path.basename(os.path.dirname(bin_path)), p)
+
+    @parameterized.expand([[None, 'FP16-INT8'], [['FP16-INT8'], None],
+                           [['FP16-INT8'], 'FP16-INT8']])
+    def test_load_intel_INT8(self, p, pr):
+        model_name = 'bert-large-uncased-whole-word-masking-squad-int8-0001'
```

Comment:
This seems like a poor choice for a test case. It's one of the largest models we got, so the test will take forever to run.